client/signup/steps/website-content/contact-page-details.tsx

```diff
@@ -37,7 +36,7 @@ export function ContactPageDetails( {
 } ) {
 	const translate = useTranslate();
 	const dispatch = useDispatch();
-	const site = useSelector( getSelectedSite );
+	const site = useSelector( getSelectedSite ) ?? undefined;
```

Comment:
This prevents `site` from being `null` and makes it `undefined` instead, which is supported by `WordpressMediaUpload`.

Reply:
Or could we make `WordpressMediaUpload` support `| null` and avoid further changes? Seems like it was just fine accepting `null` before?

Reply:
Good point.